x/gamm/keeper/pool.go

```diff
@@ -297,12 +295,3 @@ func convertToCFMMPool(pool poolmanagertypes.PoolI) (types.CFMMPoolI, error) {
 	}
 	return cfmmPool, nil
 }
-
-// getPoolIdFromSharesDenom takes in a string representing a pool share denom and extracts the pool ID.
-// It returns the pool ID as a uint64 and an error if the denom is invalid.
-func getPoolIdFromSharesDenom(denom string) (uint64, error) {
-	if !strings.HasPrefix(denom, "gamm/pool/") {
-		return 0, fmt.Errorf("invalid pool share denom %s", denom)
-	}
-	return strconv.ParseUint(denom[len("gamm/pool/"):], 10, 64)
-}
```

Comment:
Note to reviewer: Removed since this is code duplication. Used the original func instead.